src/test/rgw/rgw_multi/tests.py

```diff
@@ -935,7 +935,7 @@ def test_encrypted_object_sync():
     zonegroup = realm.master_zonegroup()
     zonegroup_conns = ZonegroupConns(zonegroup)
 
-    (zone1, zone2,) = zonegroup_conns.rw_zones
+    (zone1, zone2) = zonegroup_conns.rw_zones[0:2]
```

Comment:
you can drop the tuple syntax too,  zone1, zone2 = ... will work (but either ways are fine)